src/ServiceControl/Infrastructure/SignalR/GlobalEventHandler.cs

```diff
@@ -1,27 +1,22 @@
 ﻿namespace ServiceControl.Infrastructure.SignalR
 {
-    using System.Linq;
+    using System.Collections.Generic;
     using Microsoft.AspNet.SignalR;
-    using NServiceBus;
-    using NServiceBus.Unicast.Messages;
 
-    public class GlobalEventHandler : IHandleMessages<IEvent>
+    public class GlobalEventHandler
     {
         static string[] emptyArray = new string[0];
 
-        private readonly MessageMetadataRegistry registry;
-
-        public GlobalEventHandler(MessageMetadataRegistry registry)
-        {
-            this.registry = registry;
-        }
-
-        public void Handle(IEvent @event)
+        public void Broadcast(object @event)
         {
-            var metadata = registry.GetMessageMetadata(@event.GetType());
+            var typeName = @event.GetType().Name;
+            var types = new List<string>
+            {
+                typeName
+            };
             var context = GlobalHost.ConnectionManager.GetConnectionContext<MessageStreamerConnection>();
 
-            context.Connection.Broadcast(new Envelope { Types = metadata.MessageHierarchy.Select(t=>t.Name).ToList(), Message = @event }, emptyArray)
```

Comment:
Isn't this a potential breaking change? The message metadata registry might contain multiple types but now we are only ever sending one type. Did you verify that we never ran into the multiple types scenario?

Reply:
Yeah. I've looked at the SP codebase and only found usages with concrete leaf type. I also asked the @Particular/servicepulse-maintainers but nobody recalls any usage of base types.

Reply:
Ok then I have no objections